src/twisted/newsfragments/10224.misc

```diff
@@ -0,0 +1 @@
+support Python 3.10b4
```

Comment:
Note that .misc fragments are not included in the release notes.

For my project, I prefer  to name them `.ignore` to make it clear that the fragment  will not be part of the final release notes.

Maybe have it as a `.feature`